.eslintrc

```
@@ -0,0 +1,89 @@
+{
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier", "plugin:prettier/recommended"],
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2018
+  },
+  "plugins": ["jest"],
+  "rules": {
+    "array-bracket-newline": ["error", { "multiline": true }],
+    "strict": ["error", "safe"],
+    "block-scoped-var": "error",
+    "complexity": "warn",
+    "default-case": "error",
+    "dot-notation": "warn",
+    "eqeqeq": "error",
+    "guard-for-in": "warn",
+    "linebreak-style": ["warn", "unix"],
+    "no-alert": "error",
+    "no-case-declarations": "error",
+    "no-console": "error",
+    "no-constant-condition": "error",
+    "no-continue": "warn",
+    "no-div-regex": "error",
+    "no-empty": "warn",
+    "no-empty-pattern": "error",
+    "no-implicit-coercion": "error",
+    "prefer-arrow-callback": "warn",
+    "no-labels": "error",
+    "no-loop-func": "error",
+    "no-nested-ternary": "warn",
+    "no-script-url": "error",
+    "no-warning-comments": "warn",
+    "quote-props": ["error", "as-needed"],
+    "require-yield": "error",
+    "max-nested-callbacks": ["error", 4],
+    "max-depth": ["error", 4],
+    "require-await": "error",
+    "space-before-function-paren": [
+      "error",
+      {
+        "anonymous": "never",
+        "named": "never",
+        "asyncArrow": "always"
+      }
+    ],
+    "padding-line-between-statements": [
+      "error",
+      { "blankLine": "always", "prev": "*", "next": "if" },
+      { "blankLine": "always", "prev": "*", "next": "function" },
+      { "blankLine": "always", "prev": "*", "next": "return" }
+    ],
+    "no-useless-constructor": "off",
+    "no-dupe-class-members": "off",
+    "no-unused-expressions": "off",
+    "curly": ["error", "multi-line"],
+    "object-curly-spacing": ["error", "always"],
+    "comma-dangle": ["error", "always-multiline"],
+    "@typescript-eslint/no-useless-constructor": "error",
+    "@typescript-eslint/no-unused-expressions": "error",
+    "@typescript-eslint/member-delimiter-style": [
+      "error",
+      {
+        "multiline": {
+          "delimiter": "none",
+          "requireLast": true
+        },
+        "singleline": {
+          "delimiter": "semi",
```

I had a conflict with the prettier about this config so I changed to the following and since then I don't have that problem.

```suggestion
          "delimiter": "comma",
```

---

This actually clashes on `bee-js` with prettier :(

---

don't merge then pls

---

Yeah don't worry will not merge in `bee-js`. I'll run the lint here and fix the errors though now that we have clarity on the setup :)